angelsindustries/locale/en/science.cfg

```diff
@@ -23,6 +23,19 @@ angels-war-lab-1=Warfare tech laboratory 1
 angels-war-lab-2=Warfare tech laboratory 2
 angels-war-lab-3=Warfare tech laboratory 3
 
+[entity-description]
+angels-lab-inputs=[font=default-semibold][color=1,0.9,0.75]Inputs:[/color][/font] __1__
+angels-main-lab-crash-site=Contains all the knowledge you've gathered in the past. You should try to rebuild it to it's former glory.
```

Comment:
```suggestion
angels-main-lab-crash-site=Contains all the knowledge you've gathered in the past. You should try to rebuild it to its former glory.
```

Reply:
done, but this text was from before my changes, I have just moved it within the file